apps/storefront/src/components/shared/Breadcrumbs.tsx

```diff
@@ -0,0 +1,68 @@
+import { Breadcrumb, BreadcrumbItem, BreadcrumbLink } from "@chakra-ui/react";
+import Case from "case";
+import { useMemo } from "react";
+import { Link, useLocation, useParams } from "react-router-dom";
+import useBreadcrumbItems from "../../hooks/useBreadcrumbItems";
+
+const Breadcrumbs = () => {
+  const params = useParams();
+  const location = useLocation();
+  const { items } = useBreadcrumbItems();
+
+  const parsedBreadcrumbs = useMemo(() => {
+    const [path, query] = (location.pathname ?? "").split("?");
+    const partials = path.split("/").slice(1);
+
+    return partials.map((partial, index) => {
+      const partialPath = `/${partials.slice(0, index + 1).join("/")}${
+        partial === "orders" ? "/incoming" : ""
```

Comment:
A few comments on breadcrumbs:
1. we can get rid of the order direction logic from admin here
2. can we format category names to display a little nicer? (e.g. dog-beds --> Dog Beds)
3. We'll have to rethink either the routing conventions or the breadcrumb solution because the Shop and {category} breadcrumbs are routing to 404 pages

Reply:
how about `Shop` routes to `/products` and `{catalog}` routes to `/shop/{catalog}/products`?

probably best done by refactoring our routes.tsx to use the "official" breadcrumb solution?
https://reactrouter.com/en/main/hooks/use-matches#breadcrumbs